superset-frontend/packages/superset-ui-core/test/query/types/AnnotationLayer.test.ts

```diff
@@ -93,7 +93,6 @@ describe('AnnotationLayer type guards', () => {
     },
   ];
   const recordAnnotationResult: RecordAnnotationResult = {
-    columns: ['col1', 'col2'],
```

Comment:
LOL. The `columns` here should be the same columns as the `records` which is the output of the conversion of a Pandas DataFrame to a dictionary, i.e., this should have read, 

```javascript
columns: ['a', 'b']
```